app/my_jss/LoginFunctionality/LoginView.js

```diff
@@ -0,0 +1,49 @@
+import  { AccountModel } from './AccountModel.js';
+
+const LoginView = Backbone.View.extend({
+  className: 'login-template',
+  template() {
+    const templateText = document.querySelector(`script#${this.className}`).innerText;
+    const compiled = _.template(templateText);
+    return compiled.apply(this, arguments);
+  },
+  events: {
+    'submit #formSubmit': '_Login',
+  },
+  _Login(event) {
+    const account = {
+      username: document.getElementById('email_signIn').value,
+      password: document.getElementById('password_signIn').value,
+    };
+    const self = this;
+    $.ajax( {
```

Comment:
Acum lucrez la partea asta cu userModel data, ma gandeam daca e ceva 
posibilitate sa nu folosesc Jquery ajax ci sa folosesc fetch cu post

On 06.04.2016 09:36, Andrei Sebastian Cîmpean wrote:
>
> In app/my_jss/LoginFunctionality/LoginView.js 
> <https://github.com/SOFTVISION-University/cluj-pre-js-2016/pull/123#discussion_r58657478>:
>
> > +  className: 'login-template',
> > +  template() {
> > +    const templateText = document.querySelector(`script#${this.className}`).innerText;
> > +    const compiled = _.template(templateText);
> > +    return compiled.apply(this, arguments);
> > +  },
> > +  events: {
> > +    'submit #formSubmit': '_Login',
> > +  },
> > +  _Login(event) {
> > +    const account = {
> > +      username: document.getElementById('email_signIn').value,
> > +      password: document.getElementById('password_signIn').value,
> > +    };
> > +    const self = this;
> > +    $.ajax( {
>
> view-ul nu comunică cu serverul, call-urile sunt făcute de model
>
> asta nu e o abordare OK
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/SOFTVISION-University/cluj-pre-js-2016/pull/123/files/523bc2dfbd197a5586aafaa42067d3b4f718dd0f..31431cfe0b65cceca6f552c1134f53b6d254f06d#r58657478>
>